lib/rules/valid-jsdoc.js

```diff
@@ -16,6 +16,8 @@ const doctrine = require("doctrine");
 
 module.exports = {
     meta: {
+        type: "problem",
```

Comment:
This seems like it could be a `suggestion` since a lot of its constraints are stylistic.

Reply:
I disagree here. The main goal of the rule is to ensure that parameters in the JSDoc are matched with actual parameters in the function declaration. While it's true there are some other more stylistic constraints built-in (probably incorrectly), I think the main goal is to identify a problem.

Reply:
After thinking about this some more, I've convinced myself that this should be a suggestion because it has to do with a comment rather than code.